test-toolkit/conftest.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
is this because you copied from pytorch (training)?

Reply:
yeap